### PR TITLE
seo-bots-whitelists tests

### PR DESCRIPTION
**Pull request by buixor**

### CODE REVIEW

File: postoverflows/s01-whitelist/crowdsecurity/seo-bots-whitelist.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: crowdsecurity/seo-bots-whitelist
 description: "Whitelist good search engine crawlers"
+debug: true
```
Comment (registergoofy) [inline postoverflows/s01-whitelist/crowdsecurity/seo-bots-whitelist.yaml:3]: Is this intended ?

**Review by registergoofy — COMMENTED** [review]
Is the debug intended ? Ths has fired a new version of the item...

File: postoverflows/s01-whitelist/crowdsecurity/seo-bots-whitelist.yaml (hunk 1 above)
Comment (buixor) [reply postoverflows/s01-whitelist/crowdsecurity/seo-bots-whitelist.yaml]: oh you are damn right. by the way, isn't something we might want to detect / prevent ?

**Review by registergoofy — APPROVED** [review]
lgtm

**Review by registergoofy — APPROVED** [review]
LGTM

**Pull request merged**